src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -446,6 +452,8 @@ def match_rr_to_cause_name(
         data_keys.LDL_C.PAF,
         data_keys.SBP.RELATIVE_RISK,
         data_keys.SBP.PAF,
+        data_keys.BMI.RELATIVE_RISK,
```

Comment:
Don't you just want every RR or PAF key? Why not be generic so you don't have to add each risk individually?